crates/ide/src/move_item.rs

```diff
@@ -465,6 +465,48 @@ fn main() {}
         );
     }
 
+    #[test]
+    fn test_move_fn_up() {
+        check(
+            r#"
+fn main() {
+    println!("Hello, world!");
+}
+
+fn hi$0$0(arg: usize) {}
+            "#,
+            expect![[r#"
+fn hi$0(arg: usize) {}
+
+fn main() {
+    println!("Hello, world!");
+}
+            "#]],
+            Direction::Up,
+        );
+    }
+
+    #[test]
+    fn test_move_fn_down() {
+        check(
+            r#"
+fn hi$0$0(arg: usize) {}
+
+fn main() {
+    println!("Hello, world!");
+}
+            "#,
+            expect![[r#"
+fn main() {
+    println!("Hello, world!");
+}
+
+fn hi$0(arg: usize) {}
+            "#]],
+            Direction::Down,
+        );
+    }
```

Comment:
Let's remove these tests, if the problem ended up being on the VS Code side. We try to keep the test suite minimal.

Reply:
Okay, got it.